src/state/mint/hooks.ts

```diff
@@ -160,10 +160,16 @@ export function useDerivedMintInfo(
       wrappedCurrencyAmount(currencyBAmount, chainId),
     ]
     if (pair && totalSupply && tokenAmountA && tokenAmountB) {
-      return pair.getLiquidityMinted(totalSupply, tokenAmountA, tokenAmountB)
-    } else {
-      return undefined
+      try {
+        return pair.getLiquidityMinted(totalSupply, tokenAmountA, tokenAmountB)
+      } catch (e) {
+        if (e instanceof InsufficientInputAmountError) {
```

Comment:
idt instanceof works correctly with errors in typescript (i ran into issues with it before). can you repro and this fixes it?

Reply:
ah use `isInsufficientInputAmountError`

Reply:
instanceof was working for me but switched to isIns..